mitmproxy/tools/web/filters.py

```diff
@@ -0,0 +1,37 @@
+from typing import List
+
+from mitmproxy import flowfilter
+
+
+class FiltersManager:
+    def __init__(self, view):
+        self.filters: dict[str, str] = {
+            "search": "",
+            "highlight": "",
```

Comment:
Coding style tips:

1. Don't treat "search" and "highlight" special on the back end side. Work with arbitrary filter names.
2. If filters are inactive/empty, drop them from the list. Only keep active filters in the list.